01-js-intro-and-jquery/05/js/scripts.js

```diff
@@ -0,0 +1,7 @@
+$(document).ready(function() {
+
+	window.jQuery || document.write('<script src="../js/jquery-2.1.4.min.js"><\/script>');
+	
+	$('section').fadeIn('slow');
+	
```

Comment:
**Minor:** as a code styling convention, there's no need to have a line break before closing your main function wrapper.